build/configs/imxrt1050-evk/README.md

```diff
@@ -0,0 +1,25 @@
+# IMXRT1050-EVK
+
+## Contents
+> [Information](#information)  
+> [Environment Set-up](#environment-set-up)  
+> [How to program a binary](#how-to-program-a-binary)  
+
+## Information
+
+Please refer NXP Semiconductors [homepage](https://www.nxp.com/support/developer-resources/software-development-tools/i.mx-developer-resources/i.mx-rt1050-evaluation-kit:MIMXRT1050-EVK).
+
+## Environment Set-up
+This section covers board-specific environment set-up.  
+Please set TizenRT common environment, [quick start](https://github.com/Samsung/TizenRT#quick-start), first before doing below.
```

Comment:
This tab, `## Environment Set-up` does not have information specific NXP and it is at top of TizenRT already. Let's remove this tab.
But don't you need to say a USB rule??
Could you refer [ARTIK's README](https://github.com/Samsung/TizenRT/blob/master/build/configs/artik053/README.md#add-usb-device-rules) and [Cypress's](https://github.com/Samsung/TizenRT/blob/master/build/configs/cy4390x/README.md#register-cypress-usb-device-rule)?

Reply:
I removed this tab so that it is not neccessary
Please check it